src/CommunityToolkit.Maui.Core/Views/Popup/MauiPopup.macios.cs

```diff
@@ -49,21 +49,32 @@ public override void ViewDidLayoutSubviews()
 
 		View.Superview.Layer.CornerRadius = 0.0f;
 		View.Superview.Layer.MasksToBounds = false;
-		PresentationController?.ContainerView?.Subviews.OfType<UIImageView>().FirstOrDefault()?.RemoveFromSuperview();
+
+		SetShadowView(PresentationController!.ContainerView);
```

Comment:
please do not use null-forgiving operator

Reply:
@VladislavAntonyuk , Thank you for your review.
Removed null-forgiving operator.